pkgs/tools/system/tree/default.nix

```diff
@@ -30,12 +30,12 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     sed -i Makefile -e 's|^OBJS=|OBJS=$(EXTRA_OBJS) |'
+    makeFlags+=("CC=$CC")
```

Comment:
Shouldn't this be makeFlagsArray?

Reply:
uff, yes it should. I've expected it to fail if it's empty, but apparently it doesn't do that.

Reply:
https://github.com/NixOS/nixpkgs/pull/145780

Reply:
This should work out of preConfigure:
```
makeFlags = [ "CC:=$(CC)" ];
```

Reply:
> This should work out of preConfigure:
> 
> ```
> makeFlags = [ "CC:=$(CC)" ];
> ```

I tried that initially and it did also not work.